### PR TITLE
Script for manually running perf tests against DC/OS

### PR DESCRIPTION
**Pull request by wavesoft**
This commit introduces the `manual_run_dcos.sh` that can be used to run
the performance tests against an already deployed DC/OS cluster.

_Part of JIRA: DCOS-20295_

### CODE REVIEW

File: tests/performance/manual_run_dcos.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Comment (jeschkies) [inline tests/performance/manual_run_dcos.sh]: Out of curiosity, did you run [shellcheck](https://github.com/koalaman/shellcheck) against this script?

File: tests/performance/config/perf-driver/environments/target-cluster-custom.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# ----------------------------------------------------------- #
+# This fragment instructs the driver to use the marathon      #
+# exposed from an enterprise cluster. In contrast to other    #
+# cluster targets it requires the auth token to be given by   #
+# the user.                                                   #
+# ----------------------------------------------------------- #
+
+# Global test configuration
+# ===========================
+config:
```
Comment (jeschkies) [inline tests/performance/config/perf-driver/environments/target-cluster-custom.yml]: Is this already interpreted by the per driver? Where is its implementation?

**Comment by jeschkies** [comment]
@wavesoft, what is the state of this diff?

File: tests/performance/manual_run_dcos.sh (hunk 1 above)
Comment (wavesoft) [reply tests/performance/manual_run_dcos.sh]: Nope, but did now and applied some interesting fixes 😄

File: tests/performance/config/perf-driver/environments/target-cluster-custom.yml (hunk 2 above)
Comment (wavesoft) [reply tests/performance/config/perf-driver/environments/target-cluster-custom.yml]: Yes, this is part of the [standard driver configuration](http://dcos-performance-test-driver.readthedocs.io/en/latest/config/Readme.html). 

The driver itself is supposed to be available in the user's environment before running the manual script.

**Comment by wavesoft** [comment]
Key @jeschkies , for some reason your comments slipped through my e-mail filters 😄 I addressed your comments so it should be ready to merge.

I have used this script to run the tests on my local infrastructure, so the actual PR is not needed. I just thought it would be nice to have it on our toolbox.

**Pull request merged**